tests/subsys/tracing/tracing_api/testcase.yaml

```diff
@@ -0,0 +1,10 @@
+tests:
+  tracing.transport.uart.async.test:
+    platform_allow: qemu_x86
+    extra_args: CONF_FILE="prj_uart.conf"
```

Comment:
I think this is a common file, you can rename prj_uart.conf as prj.conf, then don't need to use extra_args

Reply:
Hi @chen-png , I have updated the code, thanks.